examples/LAPACK/foo.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> // std::cout
+#include <cblas.h>
```

Comment:
When I compile OpenBLAS on Linux, it provides a Lapack as well if gfortran is detected.